keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -176,36 +174,46 @@ Since images can be pre-loaded, loaded outside the `kubelet` process, and
 garbage collected.. the list of images that required authentication in `kubelet`
 will not be a source of truth for how all images were pulled that are in the
 container runtime cache. To mitigate, images can be garbage collected at boot.
-And for alpha, we will not persist ensure metadata across reboot of host, and restart
+And we will persist ensure metadata across reboot of host, and restart
 of kubelet, and possibly look at a way to add ensure metadata for images loaded
 outside of kubelet. In beta we will add a switch to enable re-auth on boot for
 admins seeking that instead of having to garbage collect where they do not use
 or expect preloaded images since boot.
 
+
 ## Design Details
 
-Kubelet will track, in memory, a hash map for the credentials that were successfully
-used to pull an image.
+Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image.
```

Comment:
nit: could we drop hash map in this update as it is an implementation detail? We could refer to it as `pulled image auth cache` or something similar.